static/js/app.js

```diff
@@ -180,6 +227,13 @@ function deleteAliasConfirmation(submitEvent) {
 	});
 }
 
+function submitEmailForwardingPrefsForm(submitEvent) {
+  submitEvent.preventDefault();
+  const isUserPremium = document.querySelector("body").classList.contains("is-premium");
+  const form = submitEvent.target.closest("form");
```

Comment:
`changeEvent` works great here. I was using the previous function, thus the `submitEvent` argument name. 

Originally, the toggle was a submit button, rather than the radio buttons we're grafting in. 

Updated in e98ce2ad296909333271ce2460aebb63c189b09f